### PR TITLE
Remove market status from MarketDisputed, MarketReported and MarketResolved event.

### LINKED ISSUE #655: [prediction_markets] Remove `MarketStatus` field from market status transition events like `MarketResolved`

### PR DESCRIPTION
**Pull request by vivekvpandya**
Fixes #655

### CODE REVIEW

**Comment by vivekvpandya** [comment]
It may need some updates in subsquid where there events are digested.

File: zrml/prediction-markets/src/lib.rs [hunk 1]
```diff
@@ -1228,15 +1220,15 @@ mod pallet {
...
-        MarketDisputed(MarketIdOf<T>, MarketStatus, MarketDispute<T::AccountId, T::BlockNumber>),
+        MarketDisputed(MarketIdOf<T>, MarketDispute<T::AccountId, T::BlockNumber>),
         /// An advised market has ended before it was approved or rejected. \[market_id\]
         MarketExpired(MarketIdOf<T>),
         /// A pending market has been rejected as invalid. \[market_id\]
         MarketRejected(MarketIdOf<T>),
         /// A market has been reported on \[market_id, new_market_status, reported_outcome\]
-        MarketReported(MarketIdOf<T>, MarketStatus, Report<T::AccountId, T::BlockNumber>),
+        MarketReported(MarketIdOf<T>, Report<T::AccountId, T::BlockNumber>),
         /// A market has been resolved \[market_id, new_market_status, real_outcome\]
```
Comment (Chralt98) [inline zrml/prediction-markets/src/lib.rs:1230]:
```suggestion
        /// A market has been resolved \[market_id, real_outcome\]
```

File: zrml/prediction-markets/src/lib.rs [hunk 2]
```diff
@@ -1228,15 +1220,15 @@ mod pallet {
         /// A market has been closed \[market_id\]
         MarketClosed(MarketIdOf<T>),
         /// A market has been disputed \[market_id, new_market_status, new_outcome\]
```
Comment (Chralt98) [inline zrml/prediction-markets/src/lib.rs:1222]:
```suggestion
        /// A market has been disputed \[market_id, new_outcome\]
```

File: zrml/prediction-markets/src/lib.rs [hunk 3]
```diff
@@ -1228,15 +1220,15 @@ mod pallet {
...
-        MarketDisputed(MarketIdOf<T>, MarketStatus, MarketDispute<T::AccountId, T::BlockNumber>),
+        MarketDisputed(MarketIdOf<T>, MarketDispute<T::AccountId, T::BlockNumber>),
         /// An advised market has ended before it was approved or rejected. \[market_id\]
         MarketExpired(MarketIdOf<T>),
         /// A pending market has been rejected as invalid. \[market_id\]
         MarketRejected(MarketIdOf<T>),
         /// A market has been reported on \[market_id, new_market_status, reported_outcome\]
```
Comment (Chralt98) [inline zrml/prediction-markets/src/lib.rs:1228]:
```suggestion
        /// A market has been reported on \[market_id, reported_outcome\]
```

**Review by Chralt98 — APPROVED** [review]
LGTM, event parameters reduced ✅ .

**Comment by sea212** [comment]
This is a won't fix #655

**Pull request closed without merging**